README.md

```diff
@@ -60,4 +57,70 @@ These tables are an addition to the existing PEtab format to describe the covari
 
 - No table
 
-  If no table is provided, only the variances, but no covariances are estimated.
\ No newline at end of file
+  If no table is provided, only the variances, but no covariances are estimated.
+  
+## Measurement table
+
+In addition to the existing descriptions of fixed effects, the NLME extension adds the following columns:
+
+- `$covariateId` one column per distinct covariate
+  
+## Covariate table
+
+This table describes the addition of covariate table in the PEtab NLME extension.
+
+### Detailed field description
+
+- `covariateId` [STRING]
+
+   ID of the covariate.
+
+- `covariateParameter` [STRING]
+
+   Defines the parameter where the covariate effect (COVEFF) is modelled. It must be a `parameterId` defined in the parameter table. The COVEFF would be multiplied with the `parameterId`.
+
+-  `covariateFormula` [STRING: 'exclude' or 'fractional' or 'power' or free text formula]
+
+   Covariate effect function as plain text formula expression. The user can define its own formula or can use predefined models: 'exclude' or 'fractional' or 'power'.
```

Comment:
The interpretation is correct.

The behavior for `exclude` would be that, in this case, SEX is not having an impact at all in the model (regardless of the actual category male or female). So, the effect of SEX would be none for both categories.

I suggested this `exclude` behavior thinking about testing the inclusion of different covariates in the model to try to have it in a easy manner for the user.

But, as I indicate in another comment in this PR, the approach about this extension is something we need to discuss 😄

Reply:
Got it.

> the approach about this extension is something we need to discuss

How about my suggestion above of using a `select` column, instead of an `estimate` column?